src/App.js

```diff
@@ -12,19 +12,13 @@ import "./App.css";
 
 function App() {
   const [game, setGame] = useState(<Game />);
-  const [currentUser, setCurrentUser] = useState(null);
+  const [currentUser, setCurrentUser] = useState("guest");
```

Comment:
Using null instead of "guest" is very convenient for conditional rendering and has been used in this way - changing this might break some features (e.g., making account page inaccessible to guests)